accounts/addproofer.php

```diff
@@ -184,6 +186,11 @@
 echo "  </td>";
 echo "</tr>\n";
 
+echo "<tr id='referrer_details'>";
+echo "  <th>" . _("Other Explanation") . ":</th>";
```

Comment:
The more I look at it, the more awkward "Other Explanation" sounds to me, but I don't think "Other Details" sounds any better. Can we make the label just "Other" and, in the text field add a placeholder attribute with something like "Please provide a short description"? Or possibly do away with the label altogether, and make that whole row into text input with placeholder text describing what's needed?

Reply:
I think Other details makes sense. I don't like putting instruction or some sort of label in the placeholder for various reasons: https://www.nngroup.com/articles/form-design-placeholders/.

Reply:
Interesting article, and I can definitely agree with their points, though I guess I hadn't given placeholder text a lot of thought, because I've actually appreciated it when I've seen it. And we do have it in other places in the code (at least some of it at my instigation).

Part of my concern is just wondering if we need to give more of a hint than "Other Details". Should there be a comma? -- it seems to me that there's a difference between "Other Details" and "Other, Details". And yes, I'm splitting hairs.

Reply:
I like "Other Details" with the comma, personally.

Reply:
Updated. Thanks for the advice. I am so bad at these kinds of decisions.

Reply:
It's a group effort -- we all need proofreaders!